test/files/pos/byname-implicits-31/Main_2.scala

```diff
@@ -1,3 +1,4 @@
+// scalac: -Ycheck:all
```

Comment:
Just verified that it has to be `-Ycheck:_`. Otherwise, you get a warning, which doesn't fail the test but also presumably doesn't verify what was meant to be verified. A `pos` test is a very low bar.